sessionReporter.ts

```diff
@@ -66,7 +66,7 @@ function formatGroupedByResults(testAndResults: Array<TestAndResult>) {
 }
 
 class SessionReporter implements Reporter {
-  private printTestConsole = false;
+  private printTestConsole = true;
```

Comment:
this should maybe be a proces.env variable

Reply:
ok updated it to use .env variable